src/main/java/pl/edu/agh/timekeeper/controller/AddRestrictionController.java

```diff
@@ -122,21 +127,20 @@ private void okClicked(ActionEvent actionEvent) {
         Application app = appOpt.orElseGet(() -> new Application(applicationPath, applicationPath));
 
         if (app.getRestriction() != null) {
-            // TODO: show message "Restriction for this application already exists"
             System.out.println("Restriction for this application already exists");
```

Comment:
tutaj było:
// TODO: show message "Restriction with given name already exists"
i jest usunięte a nie widzę nigdzie taska żeby to zrobić, chyba że istnieje task w którym ma to być zrobione lub dałeś gdzieś komentarz w tasku żeby nie zapomnieć

Reply:
A przy okazji to chyba nie ten task ale jak coś to gdy doda się taką samą appke to się wawala program z okno z dodawania zostaje